docs/en-US/guide/dev-guide.md

```diff
@@ -20,7 +20,7 @@ the project will install all dependencies.
 With command
 
 ```shell
-pnpm docs:dev
+pnpm docs
```

Comment:
Executing `pnpm docs` will cause problems, opening the page `https://www.npmjs.com/package/undefined` in the browser.